plerkle_snapshot/src/append_vec.rs

```diff
@@ -108,14 +108,17 @@ pub struct StoredAccountMeta<'a> {
 
 impl<'a> StoredAccountMeta<'a> {
     /// Return a new Account by copying all the data referenced by the `StoredAccountMeta`.
-    pub fn clone_account(&self) -> AccountSharedData {
-        AccountSharedData::from(Account {
-            lamports: self.account_meta.lamports,
-            owner: self.account_meta.owner,
-            executable: self.account_meta.executable,
-            rent_epoch: self.account_meta.rent_epoch,
-            data: self.data.to_vec(),
-        })
+    pub fn clone_account(&self) -> (StoredMeta, AccountSharedData) {
```

Comment:
It looks like a good candidate for `From` trait tbh

Reply:
i believe it's only used in our ETL, and while code-wise it'd be right to do, don't see it to be a big improvement in our case